### PR TITLE
Features/hero

### PR DESCRIPTION
**Pull request by QNeee**
hero

### CODE REVIEW

File: src/partials/main/hero.html [hunk 1]
```diff
@@ -1,3 +1,39 @@
-<section>
+<section class="section">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:1]: Додати id="#hero"

Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:1]: hero

File: src/partials/main/hero.html [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+  <div class="container">
+    <div class="hero-flex">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:3]: hero__flex

File: src/partials/main/hero.html [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+      <div class="hero-description">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:4]: hero__description

File: src/partials/main/hero.html [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+        <ul class="description-list">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:5]: hero__list

File: src/partials/main/hero.html [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+          <li class="description-list__item description-list__item--first">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:6]: hero__description-list

Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:6]: hero__description-item hero__description-item--first

File: src/partials/main/hero.html [hunk 6]
```diff
@@ -1,3 +1,39 @@
...
+            <p class="section__text description__text">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:7]: hero__description-text

File: src/partials/main/hero.html [hunk 7]
```diff
@@ -1,3 +1,39 @@
...
+              Noise cancellation headphones that are capable for a lot of things
+            </p>
+          </li>
+          <li class="description-list__item description-list__item--second">
+            <p class="section__text description__text">
+              Up to 20 hours of playback
+            </p>
+          </li>
+        </ul>
+        <div class="hero">
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:17]: hero__wrap

File: src/partials/main/hero.html [hunk 8]
```diff
@@ -1,3 +1,39 @@
...
+          <h1 class="hero__title">Headphones Bose NC 700</h1>
+          <button type="button" class="button button--primary">
+            order
+          </button>
+        </div>
+      </div>
+      <div>
```
Comment (Zaimak-Andrii) [inline src/partials/main/hero.html:24]: hero__thumb

**Review by Zaimak-Andrii — CHANGES_REQUESTED** [review]

File: src/partials/main/hero.html (hunk 1 above)
Comment (QNeee) [reply src/partials/main/hero.html]: +

Comment (QNeee) [reply src/partials/main/hero.html]: +

File: src/partials/main/hero.html (hunk 2 above)
Comment (QNeee) [reply src/partials/main/hero.html]: +

File: src/partials/main/hero.html (hunk 3 above)
Comment (QNeee) [reply src/partials/main/hero.html]: +

File: src/partials/main/hero.html (hunk 5 above)
Comment (QNeee) [reply src/partials/main/hero.html]: +

File: src/partials/main/hero.html (hunk 6 above)
Comment (QNeee) [reply src/partials/main/hero.html:7]: +

File: src/partials/main/hero.html (hunk 7 above)
Comment (QNeee) [reply src/partials/main/hero.html]: +

File: src/partials/main/hero.html (hunk 8 above)
Comment (QNeee) [reply src/partials/main/hero.html:22]: +

File: src/sass/layouts/main/_hero.scss [hunk 9]
```diff
@@ -1,14 +1,7 @@
...
-        height: 302px;
 
-    }
-}
 
-.hero-flex {
+.hero__flex {
     @media screen and (min-width:$tablet) {
         display: flex;
     }
```
Comment (Zaimak-Andrii) [inline src/sass/layouts/main/_hero.scss:7]: @media screen and (min-width:$desktop) {
        padding-top: 15px;
    }  

delete

**Review by Zaimak-Andrii — CHANGES_REQUESTED** [review]

File: src/sass/layouts/main/_hero.scss [hunk 10]
```diff
@@ -1,14 +1,7 @@
 // Володимир
-.hero {
```
[the remark this answers is not in the document]
Comment (QNeee) [reply src/sass/layouts/main/_hero.scss]: +

File: src/sass/layouts/main/_hero.scss (hunk 9 above)
Comment (QNeee) [reply src/sass/layouts/main/_hero.scss:12]: +

**Review by Zaimak-Andrii — APPROVED** [review]

**Pull request merged**